README.md

```diff
@@ -20,8 +20,6 @@ Verus is under *active development*. Features may be broken and/or missing, and
 the documentation is still incomplete. If you want to try Verus, please be
 prepared to ask for help in the [💬 Zulip](https://verus-lang.zulipchat.com/).
 
-The Verus community has published a number of research papers, and there are a variety of industry and academic projects using Verus. You can find a list on our <a href="https://verus-lang.github.io/verus/publications-and-projects/">publications and projects</a> page. If you're using Verus please consider adding your project to that page (see the instructions there).
```

Comment:
Even if it leads to a little bit of repetition, I'd like to preserve this paragraph. It's a more direct link to Verus-related publications and projects, which aren't necessarily just example code that people are looking for as a way to "see Verus used in action".

Reply:
Done.